templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/porter.yaml

```diff
@@ -82,6 +82,19 @@ outputs:
       - install
       - start
       - stop
+      - reset_password
+  - name: vm_username
+    type: string
+    applyTo:
+      - reset_password
+  - name: vm_password_secret_name
+    type: string
+    applyTo:
+      - reset_password
+  - name: keyvault_name
+    type: string
+    applyTo:
+      - reset_password
```

Comment:
Per [this ](https://github.com/getporter/porter/discussions/2230#discussioncomment-3093235) looks like you can remove the bundle level outputs and everything will still work.
As we don't need to return and expose these values... worth a try, right?

Reply:
Ah that's good to know - have removed the unnecessary outputs